libs/core/algorithms/include/hpx/parallel/util/detail/algorithm_result.hpp

```diff
@@ -21,6 +21,8 @@
 namespace hpx { namespace parallel { namespace util { namespace detail {
 
     ///////////////////////////////////////////////////////////////////////////
+    /// \struct algorithm_result<ExPolicy, FwdIter>
```

Comment:
I am not sure yet how to document the `algorithm_result` and all its variations because it causes multiple warnings. I added it anyway to discuss it in the next meeting.